maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractMapVersionsPhase.java

```diff
@@ -39,6 +43,7 @@
 import org.apache.maven.shared.release.versions.VersionParseException;
 import org.codehaus.plexus.components.interactivity.Prompter;
 import org.codehaus.plexus.components.interactivity.PrompterException;
+import org.codehaus.plexus.util.SelectorUtils;
```

Comment:
I'd like to avoid addition al plexus dependencies if at all possible.

Reply:
I think this should be handled as an independent issue, as it is currently being used in multiple places

Reply:
Removing existing usages is a separate issue. Not adding new usages is still important for each PR.

Reply:
Removed use of `SelectorUtils`